src/ServiceControl.AcceptanceTests/SagaAudit/When_multiple_messages_are_emitted_by_a_saga.cs

```diff
@@ -23,7 +24,7 @@ public void All_outgoing_message_intents_should_be_captured()
                 .WithEndpoint<ManagementEndpoint>(c => c.AppConfig(PathToAppConfig))
                 .WithEndpoint<EndpointThatIsHostingTheSaga>(b => b.Given((bus, c) => bus.SendLocal(new MessageInitiatingSaga())))
                 .Done(c => c.Done && TryGet("/api/sagas/" + c.SagaId, out sagaHistory))
-                .Run(TimeSpan.FromSeconds(40));
+                .Run(TimeSpan.FromSeconds(400));
```

Comment:
Het @SimonCropp  - Increased from 40 seconds to 6 and half minutes. Does it really new to wait that long to run the test?